dshr/dshr_mod.F90

```diff
@@ -510,6 +523,30 @@ subroutine dshr_set_runclock(gcomp, rc)
        call ESMF_AlarmSet(restart_alarm, clock=mclock, rc=rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
+       !----------------
```

Comment:
Same also in here. It would be nice to check `isPresent=isPresent` and `isSet=isSet`to make a default value for them. This was not required before.

Reply:
So, btw, we could get rid of adding extra attributes to `nems.configure`.

Reply:
my understanding is that we will add these to nems.configure

Reply:
yes. it is better than having defaults.